EjerciciosDiaUno.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+public class Class1
```

Comment:
Que es class1? eso esta incorrecto, por favor hemos hablado varias veces sobre el nombramiento adecuado de las clases, metodos y variables, corregirlo!

```suggestion
public class EjerciciosDiaUno
```

Reply:
Realizare una reestructuración del Ejercicio # 1 de .Net a lo largo y ancho del código, nombramientos de clases, métodos y variables, incluso trabajarlo en un solo idioma; aprovechare que me funciona. Eliminare lo que no haga falta dentro del código como son los Main, y también nombrar a todos los métodos públicos.